From ceb2f74dbac88f4c038d5d773ba5c21cb6a612b8 Mon Sep 17 00:00:00 2001
From: Robert Kanter <rkanter@apache.org>
Date: Wed, 13 Apr 2016 17:45:18 -0700
Subject: [PATCH 1423/2848] YARN-4541. Change log message in
 LocalizedResource#handle() to DEBUG (rchiang via
 rkanter)

(cherry picked from commit 0d9194df00fd68bfb7a8ba504b0cddd7d7c69b8a)

Change-Id: I591db3ddc4047cdf21ebf90db8247698a6f0540e
---
 .../localizer/LocalizedResource.java               |    8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/localizer/LocalizedResource.java b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/localizer/LocalizedResource.java
index e2d0fe1..a5f0337 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/localizer/LocalizedResource.java
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/localizer/LocalizedResource.java
@@ -200,9 +200,11 @@ public void handle(ResourceEvent event) {
         LOG.warn("Can't handle this event at current state", e);
       }
       if (oldState != newState) {
-        LOG.info("Resource " + resourcePath + (localPath != null ? 
-          "(->" + localPath + ")": "") + " transitioned from " + oldState
-            + " to " + newState);
+        if (LOG.isDebugEnabled()) {
+          LOG.debug("Resource " + resourcePath + (localPath != null ?
+              "(->" + localPath + ")": "") + " transitioned from " + oldState
+              + " to " + newState);
+        }
       }
     } finally {
       this.writeLock.unlock();
-- 
1.7.9.5

